lib/Wikia/src/Domain/User/Preferences/UserPreferences.php

```diff
@@ -80,4 +83,28 @@ public function getGlobalPreferences() {
 	public function getLocalPreferences() {
 		return $this->localPreferences;
 	}
+
+	/**
+	 * This toggle is to flag the object as being not suitable for saving.
+	 *
+	 * @param boolean state true to set the preferences to read only
+	 */
+	public function setReadOnly( $state ) {
+		$this->readOnly = $state;
+		return $this;
+	}
+
+	/**
+	 * @return boolean the read only state
+	 */
+	public function getReadOnly() {
```

Comment:
I just added that so we have consistent `get`/`set`. But you are right I think we should prefer the usage of `isReadOnly` because it's more intuitive. I went ahead and yanked it in 2a74fe7.